Source/Fuse.Common/NameValuePair.uno

```diff
@@ -8,10 +8,13 @@ namespace Fuse
 		Implements `IObject`, which means the `NameValuePair` can be viewed as an object
 		with a single property.
 
+		Implements `IArray`, which means the `NameValuePair` can be viewed as an array
+		with a single element.
+
 		An `IArray` containing some `NameValuePairs` can be converted to an `IObject`
 		implementation containing all those properties using the `ObjectFromArray` method.
 	*/
-	public sealed class NameValuePair : IObject
+	public sealed class NameValuePair : IObject, IArray
```

Comment:
I'm not sure this is actually needed. I don't think the issue requires a single NameValuePair to behave as an array. The implementation also feels wrong as it strips away information, you just get the value, not the key.